llvm_mode/LLVMInsTrim.so.cc

```diff
@@ -243,7 +243,7 @@ struct InsTrim : public ModulePass {
     for (unsigned I = 0; I < PrevLocSize - 1; ++I)
       PrevLocShuffle.push_back(ConstantInt::get(Int32Ty, I));
 
-    for (unsigned I = PrevLocSize; I < PrevLocVecSize; ++I)
+    for (int I = PrevLocSize; I < PrevLocVecSize; ++I)
```

Comment:
Do not understand, what was wrong with it? Line 243 is using `unsigned` as well.

Reply:
`PrevLocVecSize` is using a signed int, so this compares signed to unsigned, potentially a cause for bugs.
`PrevLocSize` however (compared to in line 243) is a unsigned int as well.
Maybe `PrevLocVecSize` should be `unsinged` instead?

Reply:
they could all be unsigned I think. I think I was the one doing that change because it triggered a compiler warning. dunno why no other line triggered ...